packages/components/src/presentation-components/properties/inputs/ItemsLoader.tsx

```diff
@@ -0,0 +1,113 @@
+/*---------------------------------------------------------------------------------------------
+ * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+ * See LICENSE.md in the project root for license terms and full copyright notice.
+ *--------------------------------------------------------------------------------------------*/
+
+import { UNIQUE_PROPERTY_VALUES_BATCH_SIZE } from "./UseUniquePropertyValuesLoader";
```

Comment:
I think items batch size should be defined in this file and used in unique value and navigation target loader/combobox

Reply:
Moved